episodes/vaccine-comparisons.Rmd

```diff
@@ -0,0 +1,968 @@
+                                            "vaccinated",
+                                          by_group = FALSE)
+vaccinate_02_infections <- new_infections(output_vaccinate_02,
+                                          compartments_from_susceptible =
+                                            "vaccinated",
+                                          by_group = FALSE)
+
+# create intervention_type column for plotting
+baseline_infections$intervention_type <- "baseline"
+vaccinate_01_infections$intervention_type <- "vaccinate 0.001"
+vaccinate_02_infections$intervention_type <- "vaccinate 0.01"
+infections <- rbind(baseline_infections,
+                    vaccinate_01_infections,
+                    vaccinate_02_infections)
+
+
+infections %>%
+  ggplot() +
+  ggtitle("New infections") +
+  geom_line(
+            aes(time, new_infections, colour = intervention_type,
+              linetype = intervention_type
+            ),
+            linewidth = 1) +
+  scale_y_continuous(
+    labels = scales::comma
+  ) +
+  geom_vline(
+    xintercept = c(
+      vaccinate_01$time_begin,
+      vaccinate_01$time_end
+    ),
+    linetype = 2
+  ) +
+  theme_bw() +
+  labs(
+    x = "Simulation time (days)",
+    y = "Individuals"
+  )
+```
+
+### R code
+
+```{r, echo = TRUE, eval = FALSE}
+# calculate new infections
+baseline_infections <- new_infections(output_baseline,
+                                      compartments_from_susceptible =
+                                        "vaccinated",
+                                      by_group = FALSE)
+vaccinate_01_infections <- new_infections(output_vaccinate_01,
+                                          compartments_from_susceptible =
+                                            "vaccinated",
+                                          by_group = FALSE)
+vaccinate_02_infections <- new_infections(output_vaccinate_02,
+                                          compartments_from_susceptible =
+                                            "vaccinated",
+                                          by_group = FALSE)
+
+# create intervention_type column for plotting
+baseline_infections$intervention_type <- "baseline"
+vaccinate_01_infections$intervention_type  <- "vaccinate 0.001"
+vaccinate_02_infections$intervention_type  <- "vaccinate 0.01"
+infections <- rbind(baseline_infections,
+                    vaccinate_01_infections,
+                    vaccinate_02_infections)
+
+
+infections %>%
+  ggplot() +
+  ggtitle("New infections") +
+  geom_line(
+            aes(time, new_infections, colour = intervention_type,
+              linetype = intervention_type
+            ),
+            linewidth = 1) +
+  scale_y_continuous(
+    labels = scales::comma
+  ) +
+  geom_vline(
+    xintercept = c(
+      vaccinate_01$time_begin,
+      vaccinate_01$time_end
+    ),
+    linetype = 2
+  ) +
+  theme_bw() +
+  labs(
+    x = "Simulation time (days)",
+    y = "Individuals"
+  )
```

Comment:
Would it be useful to also output a quick table comparing cumulative infections under each scenario?